roles/prereqs/prerequisite/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+prerequisite_package_name: ''
```

Comment:
Naming a role `prereqs/prequisites` is sort of redunant.  I guess this would be `prereqs/packages`, except it's empty right now, so maybe we should just drop it until we actually need it.